Formula/linux-pam.rb

```diff
@@ -0,0 +1,41 @@
+class LinuxPam < Formula
+  desc "Pluggable Authentication Modules for Linux"
+  homepage "http://www.linux-pam.org"
+  url "https://github.com/linux-pam/linux-pam/releases/download/v1.5.1/Linux-PAM-1.5.1.tar.xz"
+  sha256 "201d40730b1135b1b3cdea09f2c28ac634d73181ccd0172ceddee3649c5792fc"
+  license any_of: ["BSD-3-Clause", "GPL-1.0-only"]
+  head "https://github.com/linux-pam/linux-pam.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "berkeley-db"
+  depends_on "libprelude"
+  depends_on "libtirpc"
+  depends_on :linux
+
+  skip_clean :la, "etc"
```

Comment:
What's the `etc` here? Should that stuff not lie outside of the Cellar?

Reply:
I don't know, I did not write that code, it has been there since a long time in linuxbrew-core.
With the current setup, here is the result:
```
➜  Formula git:(master) ll /home/linuxbrew/.linuxbrew/Cellar/linux-pam/1.4.0_1/.bottle/etc/
total 8.0K
-rw-r--r-- 1 root root   97 Jun  8  2020 environment
drwxr-xr-x 4 root root 4.0K Jun  8  2020 security
➜  Formula git:(master) ll /home/linuxbrew/.linuxbrew/Cellar/linux-pam/1.4.0_1/.bottle/etc/security
total 44K
-rw-r--r-- 1 root root 4.5K Jun  8  2020 access.conf
-rw-r--r-- 1 root root 2.2K Jun  8  2020 faillock.conf
-rw-r--r-- 1 root root 3.6K Jun  8  2020 group.conf
-rw-r--r-- 1 root root 1.8K Jun  8  2020 limits.conf
drwxr-xr-x 2 root root 4.0K Jun  8  2020 limits.d
-rw-r--r-- 1 root root 1.6K Jun  8  2020 namespace.conf
drwxr-xr-x 2 root root 4.0K Jun  8  2020 namespace.d
-rwxr-xr-x 1 root root 1016 Jun  8  2020 namespace.init
-rw-r--r-- 1 root root 3.0K Jun  8  2020 pam_env.conf
-rw-r--r-- 1 root root 2.2K Jun  8  2020 time.conf
```

Removing "etc", these files are installed to etc:
```
➜  Formula git:(master) ✗ ll /home/linuxbrew/.linuxbrew/etc/environment
-rw-r--r-- 1 root root 97 Mar 24 16:11 /home/linuxbrew/.linuxbrew/etc/environment
➜  Formula git:(master) ✗ ll /home/linuxbrew/.linuxbrew/etc/security/
total 44K
-rw-r--r-- 1 root root 4.5K Mar 24 16:11 access.conf
-rw-r--r-- 1 root root 2.2K Mar 24 16:11 faillock.conf
-rw-r--r-- 1 root root 3.6K Mar 24 16:11 group.conf
-rw-r--r-- 1 root root 1.8K Mar 24 16:11 limits.conf
drwxr-xr-x 2 root root 4.0K Mar 24 16:11 limits.d
-rw-r--r-- 1 root root 1.6K Mar 24 16:11 namespace.conf
drwxr-xr-x 2 root root 4.0K Mar 24 16:11 namespace.d
-rwxr-xr-x 1 root root 1016 Mar 24 16:11 namespace.init
-rw-r--r-- 1 root root 3.0K Mar 24 16:11 pam_env.conf
-rw-r--r-- 1 root root 2.2K Mar 24 16:11 time.conf
```

I'm unsure why they preferred to install stuff to .bottle/etc.

Reply:
> I'm unsure why they preferred to install stuff to .bottle/etc.

That's just the system we use for stuff that should be installed into `HOMEBREW_PREFIX` directly rather than the formula's Cellar prefix.

> I don't know, I did not write that code, it has been there since a long time in linuxbrew-core.

Probably not needed then. Similar instances have been removed from homebrew-core: https://github.com/Homebrew/homebrew-core/commit/7802f7b8293253b96d74369c9c7deaa3acea3d44

Reply:
It's probably a legacy way of handling things. I'll remove it.